src/Workspaces/CSharp/Portable/CodeGeneration/ConstructorGenerator.cs

```diff
@@ -76,12 +76,12 @@ private static MemberDeclarationSyntax LastConstructorOrField(SyntaxList<MemberD
                 var preferExpressionBody = workspace.Options.GetOption(CSharpCodeStyleOptions.PreferExpressionBodiedConstructors).Value;
                 if (preferExpressionBody)
                 {
-                    var expressionBody = declaration.Body.TryConvertToExpressionBody(options);
-                    if (expressionBody != null)
+                    if (declaration.Body.TryConvertToExpressionBody(
+                            options, out var expressionBody, out var semicolonToken))
```

Comment:
the lines were too long for me.  so i wrapped them.  Because of that, i indented to make it clear that it wasn't the start of a new expression, but a sub-nodes of the .TryConvert call.